specification/cat/_types/CatBase.ts

```diff
@@ -555,8 +556,463 @@ export enum CatDfaColumn {
    */
   version
 }
+
+export enum CatNodeColumn {
```

Comment:
Thank you for this! Note however that this enum should be marked as non-exhaustive, as values like `heap.*` are allowed. This is done by adding `/** @non_exhaustive */` as a comment. See https://github.com/elastic/elasticsearch-specification/blob/main/docs/modeling-guide.md#enum for details.

Reply:
Fixed in https://github.com/elastic/elasticsearch-specification/pull/4373